backend/src/main/java/net/vidux/camhub/discovery/RawCameraDataFactory.java

```diff
@@ -0,0 +1,48 @@
+package net.vidux.camhub.discovery;
+
+import org.springframework.stereotype.Component;
+
+@Component
+class RawCameraDataFactory {
+  private static final String REGEX = "\\t";
```

Comment:
Továbbra is azt mondom, hogy szerintem legyen ennek a konstansnak valami érthető elnevezése, mert a REGEX név annyira általános, hogy csak akkor derül ki, hogy mi a szerepe, amikor használva van. 

Ha ránézek a konstans nevére, akkor egyből tudnom kellene, hogy mi a szerepe. Pl.: RAW_CAMERA_LINE_SEPARATOR.